src/components/dev-hub/success-state.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { H3 } from './text';
+import { size } from './theme';
+import SuccessIcon from './icons/success';
+
+const SuccessContainer = styled('div')`
+    text-align: center;
+    svg {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const SuccessState = ({ children }) => (
```

Comment:
Maybe I missed this but can we reference this in the project sign up form?

Reply:
Oops! Totally forgot to do that. Thanks!